.travis.yml

```diff
@@ -24,4 +24,4 @@ before_script:
   - "flake8 ."
 
 script:
-  - "py.test --capture=sys --showlocals --tb=long foxpuppet/tests "
\ No newline at end of file
+  - "py.test --capture=sys --showlocals --tb=long --driver Firefox foxpuppet/tests "
```

Comment:
`--capture=sys`, `--showlocals` and `--tb-long` can all be useful options when debugging failures, but I don't think we need them for every Travis CI build.